packages/webcert/src/store/certificate/certificateSelectors.ts

```diff
@@ -239,3 +239,7 @@ export const getResponsibleHospName = (state: RootState): string => {
 }
 
 export const isCertificateFunctionDisabled = (state: RootState): boolean => state.ui.uiCertificate.functionDisablers.length > 0
+
+export const getCertificateId = () => (state: RootState): string => state.ui.uiCertificate.createdCertificateId
+
+export const getNavigateToCertificate = () => (state: RootState): boolean => state.ui.uiCertificate.navigateToCertificate
```

Comment:
This isn't used and can be removed, right?

Reply:
Removed getNavigateToCertificate.